config/redirects.txt

```diff
@@ -8,19 +8,21 @@
 (.*)\.md(.*) -> $1$2
 ^\/introduction\/introduction(\/|$) -> /introduction
 ^\/$ -> /introduction
-^\/installing\/gettingStarted(\/|$) -> /raspberrypi/nodejs/getting-started/
+^\/installing\/gettingStarted(\/|$) -> /raspberrypi3/nodejs/getting-started/
 ^\/installing\/gettingStarted-BBB(\/|$) -> /beaglebone/nodejs/getting-started/
 ^\/installing\/gettingStarted-Humming(\/|$) -> /hummingboard/nodejs/getting-started/
 ^\/installing\/gettingStarted-NUC(\/|$) -> /nuc/nodejs/getting-started/
 ^\/installing\/gettingStarted-Nitrogen6x(\/|$) -> /nitrogen6x/nodejs/getting-started/
 
 
 # Default redirects
-^\/getting-started -> /raspberrypi3/nodejs/getting-started
-^\/deployment\/network\/(2\.0.*|$) -> /deployment/network/2.x
-^\/deployment\/network\/1.x.x -> /deployment/network/1.x
-^\/understanding/understanding-devices\/(2\.0.*|$) -> /understanding/understanding-devices/2.x
-^\/understanding/understanding-devices\/1.x.x -> /understanding/undderstanding-devices/1.x
+^\/getting-started($) -> /raspberrypi3/nodejs/getting-started/
+^\/deployment\/network\/(2\.0.*|$) -> /2.x/deployment/network
+^\/deployment\/network\/1.x.x -> /1.x/deployment/network
+^\/understanding/understanding-devices\/(2\.0.*|$) -> /2.x/understanding/understanding-devices
+^\/understanding/understanding-devices\/1.x.x -> /1.x/understanding/understanding-devices
+^\/deployment\/network\/([1-2]\.x) -> /$1/deployment/network 
+^\/understanding\/understanding-devices\/([1-2]\.x) -> /$1/understanding/understanding-devices
```

Comment:
Is the main motivation for this that these URLs are easier to match as dynamic? I can see how that's true, but we could still handle this fairly easy in other ways (a list of dynamic URL regexes), and it feels like it makes a little more sense to me with the parameter at the end really.

Reply:
@pimterry yes, that was the main motivation. I had originally switched things the other way (so that the getting started links matched the network and resinOS links) as I agree this make more sense to have the parameter at the end. I was having trouble getting it to recognize the dynamic pages in a way that wasn't too hacky, but I have a couple of other ideas I'll try.